package.json

```
@@ -52,7 +52,10 @@
     "tests:macos:build": "cd tests && yarn build:macos",
     "tests:macos:pod:install": "cd tests && rm -f macos/Podfile.lock && cd macos && pod install",
     "tests:macos:test-cover": "cd tests && npx jet --target=macos --coverage",
-    "format:markdown": "prettier --write \"docs/**/*.md\""
+    "format:markdown": "prettier --write \"docs/**/*.md\"",
+    "example:app:bundler": "cd tests && yarn react-native start --config ./test-app/metro.config.js --reset-cache",
+    "example:app:run:ios": "cd tests && yarn react-native run-ios",
```

This proposed change -
1- works around a subtle low-level simulator crash with swizzling and Firebase Performance from 2021/2022 (!) https://github.com/firebase/firebase-ios-sdk/issues/9083 
2- allows use of ccache for faster builds if configured in user's paths (but is backwards compatible with raw Xcode if not)

```suggestion
    "example:app:run:ios": "cd tests && SIMCTL_CHILD_GULGeneratedClassDisposeDisabled=1 CC=clang CPLUSPLUS=clang++ LD=clang LDPLUSPLUS=clang++ yarn react-native run-ios",
```